sos/report/plugins/opencontrail.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2021 Mirntis, Inc., Oleksii Molchanov <omolchanov@mirantis.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenContrail(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """OpenContrail SDN
+    """
+    plugin_name = 'opencontrail'
+    profiles = ('network')
+
```

Comment:
This needs an enablement trigger, otherwise this plugin will always run.

I'd suggest something like:

```python
packages = ('opencontrail',)
containers = ('opencontrail.*',)
```

The first checks if an 'opencontrail' package is installed. Adjust this as necessary to include package names from different distros.

The second checks to see if any containers whose name matches `opencontrail.*` (thus, anything starting with opencontrail) are _currently_ running. Again, adjust the regex here to whatever the naming scheme is for these containers.

Reply:
Fixed with provided example.